### PR TITLE
DPC-545 configure container to run integration tests in it

### PR DESCRIPTION
**Pull request by whuang85**
### Fixes [DPC-545](https://jira.cms.gov/browse/DPC-545)

In Jenkins, patient everything integration test does not have access to the /tmp folder on the host (where aggregation puts it's files) so to solve this, we will run the tests in a container where we can map the /tmp folder as a volume to the container.

### Proposed Changes

- Run api integration tests in container
- Made jobTimeoutInSeconds configurable via SSM

### Change Details

Using the official maven docker image to run the mvn test command in that container instead of directly in jenkins. This allows us to map the /tmp folder of the host to the container so when patient everything runs it can access the files that aggregation generates. Also cleaned up the jobTimeoutInSeconds configuration so it can be configured via SSM.

### Security Implications

- [ ] new software dependencies

- [ ] security controls or supporting software altered

- [ ] new data stored or transmitted

- [ ] security checklist is completed for this change

- [x] requires more information or team discussion to evaluate security implications

- [x] no PHI/PII is affected by this change

### Feedback Requested

Do I need a security checklist for bringing in the new docker image although it's only ran locally (via make ci-app) and in jenkins and travis?

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -4,6 +4,7 @@ services:
 
   db:
     image: postgres
+    command: postgres -c 'max_connections=250'
```
Comment (embh) [inline docker-compose.yml:7]: Is there any downside to setting this higher than the default?

Comment (whuang85) [reply docker-compose.yml:7]: I don't believe so, when I was testing this locally, I kept running out of connections. I believe the default is 100.

File: dpc-api/src/main/resources/application.conf [hunk 2]
```diff
@@ -2,6 +2,9 @@ dpc.api {
     include "server.conf"
     include "queue.conf"
 
+    jobTimeoutInSeconds = ${?JOB_TIMEOUT_IN_SECONDS}
+    jobTimeoutInSeconds = 30
+
```
Comment (jonfulk) [inline dpc-api/src/main/resources/application.conf:7]: Is this a way of defaulting to 30 s if no value is set for `JOB_TIMEOUT_IN_SECONDS`? If there is a value set, does it just use the first one and ignore the second line?

Comment (whuang85) [reply dpc-api/src/main/resources/application.conf:7]: Yep that is exactly what those two lines do.

**Pull request merged**